scripts/src/public/routes/users.ts

```diff
@@ -127,3 +128,8 @@ export const myUserInfo = async function(req: Request, res: Response) {
 	req.params.user_id = req.context.user!.appUserId;
 	await (userInfo as any)(req as UserInfoRequest, res);
 } as any as RequestHandler;
+
+export const logoutUser = async function(req: Request, res: Response) {
+	await logoutService(req.context.user!, req.context.token!);
```

Comment:
I was thinking that we should use a different `Request` object for authenticated and non authenticated requests. The authenticated version will have context.user and token as definite fields

Reply:
that's a good idea, but it's a task/PR of it's own